README.md

```diff
@@ -1,157 +1,22 @@
-# Cloud Spanner Banking Application
+# Cloud Spanner Sample Apps
 
-This repository provides the source code for a Cloud Spanner sample financial
-application. It consists of 2 components:
-1. A gRPC server that provides application level functionality
-  (see: [service.proto](server/src/main/proto/service.proto)) using CloudSpanner
-  as the storage backend, found in the [server](server) directory
-2. A workload generator for the above, found in the [workload](workload) directory
+This repository provides the source code for Cloud Spanner sample
+applications.
 
-The goal of the sample is to provide a simple working example to jump-start
-exploring Cloud Spanner.
+These applications highlight how to develop applications to work with Cloud Spanner.
 
-## Architecture & Schema
+More details of each sample application can be found in the README for the sample
+app.
 
-### Application Components
-This is how everything fits together:
-![FinAppConnect.png](images/FinAppConnect.png)
+Sample applications:
+- [Banking](finance/README.md)
```

Comment:
```suggestion
- [Finance](finance/README.md)
```

I know we called this "banking" in the original readme but let's go with "finance" unless we change it to "banking" everywhere, including in other docs and the sample app name.

Reply:
Added in #82